charts/gardener/controlplane/charts/runtime/templates/apiserver/deployment.yaml

```diff
@@ -270,6 +270,9 @@ spec:
         - --tls-private-key-file=/etc/gardener-apiserver/srv/gardener-apiserver.key
         {{- end }}
         {{- include "gardener-apiserver.watchCacheSizes" . | indent 8 }}
+        {{- if .Values.global.apiserver.workloadIdentity.token.issuer }}
+        - --workload-identity-token-issuer={{ .Values.global.apiserver.workloadIdentity.token.issuer }}
```

Comment:
How shall `gardener-operator` configure this value when deploying `gardener-apiserver` in https://github.com/gardener/gardener/blob/13312fd8b4738e99309193d50707d0d418b5ad3a/pkg/component/gardener/apiserver/deployment.go#L92-L101

Reply:
They should be exposed via the Garden resource. The issuer should re-use the first ingress domain like the discovery server because it will be the component serving the public keys and discovery information for the workload identity issuer. 

I was leaning toward doing this integration with operator later with another PR, but thinking again, maybe I can do it today and take care to have it configured only for local setups by default, what do you think?

Reply:
I think adding this in this PR makes sense 👍🏻 However, I didn't get the point of exposing something in the Garden resource.
If it should be the same as for GDS, GOP can just configure GAPI accordingly with it - no need to expose something in the Garden?

Reply:
Sorry, I was talking about exposing configurations for the `--workload-identity-token-(min|max)-expiration` flags. `--workload-identity-token-issuer` will be computed from the ingress domains.

Reply:
Got it. For min/max values, I would propose to either stick with the defaults for now, or let GOP configure them with "best practice" values. If this does not work for users, we can always expose the fields in the Garden later. This way, we keep the API minimal for now (we followed this strategy for most other config options of Gardener components).

Reply:
Done